shared/store/action-logger.js

```diff
@@ -1,12 +1,11 @@
 // @flow
-
-import {Iterable} from 'immutable'
 import deep from 'deep-diff'
+import type {StatSink} from '../util/stats'
```

Comment:
You don't have to change it, but I don't understand, it was already separated by a blank line, so you just had to sort the top imports (and the bottom type imports) individually. No need to move the type imports up and then sort.